dir4/file1.txt

```diff
@@ -0,0 +1 @@
+This is file1 in dir4
```

Comment:
This is a comment on the code line.
Maybe you should add more lines to show how a snipped looks like

Reply:
Done.